integreat_cms/cms/templates/organizations/organization_form.html

```diff
@@ -146,6 +151,69 @@ <h3 class="heading font-bold text-black">
                                 </tbody>
                             </table>
                         </div>
+                        <div class="table-listing w-full p-2">
```

Comment:
Do you think it makes sense with regard to #2073 to already extract this to a separate django template so we can save these lines here and reuse them here and in #2073? :)

Reply:
What to you mean with "with regard to #2073"? I don't get the point.

Reply:
In #2073 we will also have a section where we you can see where the contact is being used. I'm just trying to think if we can already refactor this section in a way that will be beneficial then. I hope this makes it clearer :)

Reply:
Ah 💡 Now I got it.